check/__processes__/processes.go

```diff
@@ -0,0 +1,113 @@
+package __processes__
```

Comment:
Shall we consider renaming the package name too? It looks a bit frightening in my opinion.